editor/create_dialog.cpp

```diff
@@ -104,8 +105,19 @@ void CreateDialog::popup_create(bool p_dontclear) {
 	if (enable_rl) {
 		search_options->add_constant_override("draw_relationship_lines", 1);
 		search_options->add_color_override("relationship_line_color", rl_color);
-	} else
+	} else {
 		search_options->add_constant_override("draw_relationship_lines", 0);
+	}
+
+	bool is_replace_mode = p_replace_mode;
```

Comment:
You're defining a local `is_replace_mode` here, not setting the member var.

Reply:
Oops!